extension/war/src/main/webapp/groovy/social/webui/profile/UIUserActivitiesDisplay.gtmpl

```diff
@@ -44,39 +52,41 @@
-                                       .replace("{wikis_link}", wikis_link)
-                                       .replace("{forums_link}", forums_link)
-                                       .replace("{calendars_link}", calendars_link)
-                                       .replace("{documents_link}", documents_link);
-  def use_spaces_for_team_or_project_matters = _ctx.appRes("UIActivity.label.Use_spaces_for_team_or_project_matters");
-  def use_spaces_for_team_or_project_matters_details = _ctx.appRes("UIActivity.label.Use_spaces_for_team_or_project_matters_details")
-                                                 .replace("{spaces_link}", spaces_link);
-  def anywhere_at_anytime = _ctx.appRes("UIActivity.label.Anywhere_at_anytime");
-  def anywhere_at_anytime_details = _ctx.appRes("UIActivity.label.Anywhere_at_anytime_details")
-                                                     .replace("{iOS_App_Store_Link}", iOS_App_Store_link)
-                                                     .replace("{Google_Play}", google_play);  
+
+  //
+  def owner_name = "<a href='${ownerUri}'><strong class='text-color'>$ownerName</strong></a>";
+  def meeds_link = "<a href='https://www.meeds.io/'><strong class='text-color'>Meeds</strong></a>";
+  def meeds_association_link = "<a href='https://www.meeds.io/'><strong class='text-color'>Meeds Association</strong></a>";
+  def apis_link = "<a href='https://github.com/Meeds-io/meeds/wiki'><strong class='text-color'>APIs</strong></a>";
+  def lgpl_license_link = "<a href='https://github.com/Meeds-io/meeds/blob/develop/LICENSE'><strong  class='text-color'>the LGPL license</strong></a>";
+  def docker_link = "<a href='https://github.com/Meeds-io/meeds-docker'><strong class='text-color'>Docker</strong></a>";
+  def github_link = "<a href='https://github.com/Meeds-io/meeds'><strong>Github</strong></a>";
+
+  def welcome_title =  _ctx.appRes("UIActivity.label.Welcome_Activity_Welcome_Onboard").replace("{user full name}", owner_name);
+  def thanx_for_joing_meeds =  _ctx.appRes("UIActivity.label.Thanx_for_joing_meeds").replace("{meeds Link}",meeds_link);
+  def welcome_paragraphe_one = _ctx.appRes("UIActivity.label.paragraph_one").replace("{meeds association link}",meeds_association_link);
+  def welcome_paragraphe_two = _ctx.appRes("UIActivity.label.paragraph_two");
+  def welcome_paragraphe_three = _ctx.appRes("UIActivity.label.paragraph_three");
+  def welcome_paragraphe_four = _ctx.appRes("UIActivity.label.paragraph_four").replace("{APIs link}",apis_link);
+  def welcome_paragraphe_five = _ctx.appRes("UIActivity.label.paragraph_five").replace("{the LGPL license link}", lgpl_license_link)
+                                                                              .replace("{GitHub link}", github_link)
+                                                                              .replace("{Docker link}",docker_link);
```

Comment:
IMO it's better to split this and add this into a dedicated gtmpl file that you can include here with `_ctx.include(templatePath)` instruction. WDYT ?

Reply:
Yes, I think it is better too!!